tests/test_wcs.py

```diff
@@ -499,7 +500,7 @@ def do_local_wcs(wcs, ufunc, vfunc, name):
     wcs2 = wcs.local()
     assert wcs == wcs2, name+' local() is not == the original'
     new_origin = galsim.PositionI(123,321)
-    wcs3 = wcs.withOrigin(new_origin)
+    wcs3 = wcs.shiftOrigin(new_origin)
```

Comment:
For a LocalWCS, these are equivalent.  And IMO withOrigin is preferred, since the WCS doesn't have an origin yet.